testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/broker/AccountLinkTest.java

```diff
@@ -139,31 +138,27 @@ public void createParentChild() {
     @Test
     public void testAccountLink() {
         String childUsername = "child";
-        String childPassword = "password";
-        String childIdp = CHILD_IDP;
 
-        testAccountLink(childUsername, childPassword, childIdp);
+        testAccountLink(childUsername);
     }
 
     @Test
+    @Ignore // Ignore should be removed by https://github.com/keycloak/keycloak/issues/20441
```

Comment:
@wojnarfilip Hi Filip, please can you elaborate it more? From my point of view this test should be removed instead of ignoring it. @miquelsi @mhajas

Reply:
it applies the same as below -> testing UserStorage provider